### PR TITLE
RFR: Fixes to bootstrap scripts. 

### PR DESCRIPTION
**Pull request by lakshmi-kannan**
Fixes 

* [X] https://github.com/StackStorm/st2-packages/issues/290
* [X] https://github.com/StackStorm/st2-packages/issues/288
* [X] https://github.com/StackStorm/st2-packages/issues/287
* [X] https://github.com/StackStorm/st2-packages/issues/289 
* [x] https://github.com/StackStorm/st2-packages/issues/292

### CODE REVIEW

**Comment by DoriftoShoes** [comment]
+1

File: scripts/st2bootstrap-el6.sh [hunk 1]
```diff
@@ -89,21 +89,21 @@ get_full_pkg_versions() {
     local ST2_VER=$(yum info st2 | grep Version | awk '{print $3}' | grep ${VERSION} | sort --version-sort | tail -n 1)
     if [ -z "$ST2_VER" ]; then
       echo "Could not find requested version of st2!!!"
-      sudo apt-cache policy st2
+      sudo yum info st2
```
Comment (manasdk) [inline scripts/st2bootstrap-el6.sh]: lol!

**Comment by manasdk** [comment]
+1

**Comment by DoriftoShoes** [comment]
+1

**Pull request merged**